stdlib/VERSIONS

```diff
@@ -14,121 +14,121 @@ _dummy_threading: 2.7
 _heapq: 2.7
 _imp: 3.6
 _importlib_modulespec: 3.6
-_json: 3.6
-_markupbase: 3.6
+_json: 2.7
+_markupbase: 2.7
 _msi: 2.7
 _operator: 3.6
-_osx_support: 3.6
+_osx_support: 2.7
 _posixsubprocess: 3.6
 _py_abc: 3.7
 _pydecimal: 3.6
 _random: 2.7
 _sitebuiltins: 3.6
 _stat: 3.6
-_thread: 3.6
+_thread: 2.7
 _threading_local: 3.6
-_tkinter: 3.6
+_tkinter: 3.0
```

Comment:
Python 2 has `_tkinter` and `Tkinter` (but there are no stubs for `Tkinter`). Python 3 has `_tkinter` and `tkinter`.

```suggestion
_tkinter: 2.7
```